inc/util/any.h

```diff
@@ -158,6 +159,15 @@ struct AnyContainer {
 
     AnyContainer() noexcept { new(m_space) Container; }
 
+    // wouldn't we want to make these two ctors, as well
+    // as the assignment operators that take AnyContainer,
+    // templated (ie, arg= AnyContainer<SourcePolicy>)? 
+    // That way you could assign/construct between
+    // AnyContainers of different policies. I see something
```

Comment:
This can not be done since `AnyContainer` should adhere to the standard.  The only change that does not adhere to the standard is to allow for inheritance of `AnyContainer`, however, the standard proscribes inheriting from `std::` so this `any` interface change is immaterial with regards to standards compliance.